Library/Homebrew/tab.rb

```diff
@@ -242,6 +242,26 @@ def compiler
     super || DevelopmentTools.default_compiler
   end
 
+  def homebrew_tag
+    homebrew_version.sub(/\-\d+\-g([a-f0-9]+)(?:\-dirty)?\Z/, "")
+  end
+
+  def parsed_homebrew_version
+    return Version::NULL if homebrew_version.nil?
+    Version.new(homebrew_tag)
+  end
+
+  # Whether there is reliable runtime dependency information in the receipt.
+  def reliable_runtime_dependencies?
```

Comment:
Doing this messes with other tests that create `Tab`s because they're being created with 1.1.5 at the moment. How about we leave it for now and change it after 1.1.6 is tagged?